pkg/daemon/ceph/client/info.go

```diff
@@ -68,8 +68,8 @@ type MonInfo struct {
 // For converged clusters it will be the admin key, but external clusters will have a
 // lower-privileged key.
 type CephCred struct {
-	Username string `json:"name"`
-	Secret   string `json:"secret"`
+	OperatorUsername string `json:"name"`
```

Comment:
Is there a need to add the Operator prefix to these var names? I kinda like the simpler names without changing this.
```suggestion
	Username string `json:"name"`
```